src/commands/to-arb/hierarchical-json-to-arb.converter.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as _ from "lodash";
+
+export class HierarchicalJsonToArbConverter {
+  constructor(
+    private directoryWithJsonFiles: string,
+    private outputDir: string,
+    private lastModified: string = new Date().toISOString()
+  ) {}
+
+  convertToArb() {
+    const jsonFiles = fs
+      .readdirSync(this.directoryWithJsonFiles)
+      .filter(file => _.endsWith(file, ".json"));
+    jsonFiles.map(file => {
+      const filename = _.replace(file, ".json", ".arb");
+      const jsonInString = fs.readFileSync(
+        path.join(this.directoryWithJsonFiles, file),
+        "utf8"
+      );
+      this.convertJsonToArb(jsonInString, filename);
+    });
+  }
+
+  private convertJsonToArb(jsonInString: string, filename: string) {
+    const jsonContent = JSON.parse(jsonInString);
+    const resourceIds = Object.keys(jsonContent);
+    let resources: Record<string, any> = {};
+    resources["@@last_modified"] = this.lastModified;
+    resources["@@author"] = "arb-converter-cli";
+    resourceIds.forEach(id => {
+      resources[id] = jsonContent[id];
```

Comment:
Probably it's something conventional, but do we have to make sure that value of `jsonContent[id]` have type "string"? 
Or may be we should "flatten" it if value is an object?

Reply:
This operation is safe, since the `id` is retrieved from the JSON given initially (we are just transforming the data, not creating new fields).